include/deal.II/base/polynomials_adini.h

```diff
@@ -59,42 +64,51 @@ class PolynomialsAdini
    * function, rather than using any of the compute_value(), compute_grad() or
    * compute_grad_grad() functions, see below, in a loop over all polynomials.
    */
-
   void
-  evaluate(const Point<2> &           unit_point,
-           std::vector<double> &      values,
-           std::vector<Tensor<1, 2>> &grads,
-           std::vector<Tensor<2, 2>> &grad_grads) const;
+  evaluate(const Point<dim> &           unit_point,
+           std::vector<double> &        values,
+           std::vector<Tensor<1, dim>> &grads,
+           std::vector<Tensor<2, dim>> &grad_grads,
+           std::vector<Tensor<3, dim>> &third_derivatives,
+           std::vector<Tensor<4, dim>> &fourth_derivatives) const override;
 
   /**
    * Compute the value of the <tt>i</tt>th polynomial at <tt>unit_point</tt>.
    *
    * Consider using evaluate() instead.
    */
-
   double
-  compute_value(const unsigned int i, const Point<2> &p) const;
+  compute_value(const unsigned int i, const Point<dim> &p) const;
 
   /**
    * Compute the gradient of the <tt>i</tt>th polynomial at
    * <tt>unit_point</tt>.
    *
    * Consider using evaluate() instead.
    */
+  Tensor<1, dim>
+  compute_grad(const unsigned int i, const Point<dim> &p) const;
 
-  Tensor<1, 2>
-  compute_grad(const unsigned int i, const Point<2> &p) const;
   /**
    * Compute the second derivative (grad_grad) of the <tt>i</tt>th polynomial
    * at <tt>unit_point</tt>.
    *
    * Consider using evaluate() instead.
    */
+  Tensor<2, dim>
+  compute_grad_grad(const unsigned int i, const Point<dim> &p) const;
+
+  /**
+   * Return the name of the space, which is <tt>PolynomialSpace</tt>.
+   */
+  std::string
+  name() const override;
```

Comment:
Shouldn't this be something involving the word "Adini"?

Reply:
Ah good catch! You can see where I copied it from :stuck_out_tongue: